src/overlays/actors/ovl_En_Toto/z_en_toto.h

```diff
@@ -38,7 +38,7 @@ typedef struct EnToto {
     /* 0x2B7 */ u8 unk2B7;
     /* 0x2B8 */ EnTotoText* text;
     /* 0x2BC */ char unk2BC[0x8]; // Some z_player_lib struct size 0x8 more info from func_80122744 and func_80122760
-    /* 0x2C4 */ Actor* unk2C4;
+    /* 0x2C4 */ Actor* spotlights;
```

Comment:
This never uses Dm_Char07 specific entries. Is it fine if I just keep it as an Actor pointer, instead of an explicit Dm_Char07?

Reply:
It should probably be fine, but I'd also double check with others